buildrunner/utils.py

```diff
@@ -194,15 +195,22 @@ def write(self, output: Union[bytes, str], color=None):
         # do not colorize output to other streams
         for stream in self.streams:
             try:
-                stream.write(output)
+                if stream.closed:
+                    self.stderr.write(
+                        f'WARNING: Attempted to write to closed stream {stream}.' \
+                        f' Removed from list of streams in {__class__} and not writing {output} to {stream}.'
```

Comment:
Is this actually removed? I'm not seeing anything about removing the stream...

Reply:
awww... I took that logic out but didn't update the comment.

Reply:
I updated the warning message. I deciding I wanted the warning to be printed multiple times if it continues to try to log to a closed file. That is why I didn't remove the closed stream from the list of streams.